Data/models/DataController.swift

```diff
@@ -44,6 +40,7 @@ public class DataController: NSObject {
                 }
             } catch {
                 log.error(error)
```

Comment:
Let's add a log prefix to this, maybe then we can get someone to pull the log.
```suggestion
                log.error("Database Migration Error: \(error)")
```